lib/jekyll/configuration.rb

```diff
@@ -207,7 +207,7 @@ def read_config_files(files)
       rescue ArgumentError => err
         Jekyll.logger.warn "WARNING:", "Error reading configuration. " \
                      "Using defaults (and options)."
-        $stderr.puts err
+        warn err
```

Comment:
```text
lib/jekyll/configuration.rb:210:9: C: Style/StderrPuts: Use warn instead of $stderr.puts.
        $stderr.puts err
        ^^^^^^^^^^^^
```

Reply:
can these be changed to use Jekyll's logger instead?

Reply:
I wonder if we could just append it to the string we are already logging?

Reply:
Revisiting this, I'm not sure if we should be changing an outcome of a public method.. (..in the rare case someone's capturing `$stderr` for some purpose..)

So I think its best that we stick to `warn` in this PR or `disable` that check entirely if that's better..

Reply:
Would it be changing behavior? I'm sure our logger outputs to stderr, right?